src/chip.tsx

```diff
@@ -76,24 +101,59 @@ const Chip: React.FC<ChipProps> = ({Icon, children, id, dataAttributes, active,
                 </Box>
             </Box>
         );
-    } else {
-        const isInteractive = active !== undefined;
+    }
+    const isInteractive = active !== undefined || props.href || props.onPress || props.to;
+
+    const renderContent = () => (
+        <Box
+            className={classnames(
+                styles.chipVariants[active ? 'active' : overAlternative ? 'overAlternative' : 'default'],
+                {
+                    [styles.chipInteractiveVariants[isDarkMode ? 'dark' : 'light']]: isInteractive,
+                }
+            )}
+            paddingLeft={paddingLeft}
+            paddingRight={12}
+            {...getPrefixedDataAttributes(dataAttributes, 'Chip')}
+        >
+            {body}
+        </Box>
+    );
+
+    if (props.onPress) {
         return (
-            <Box
-                className={classnames(
-                    styles.chipVariants[active ? 'active' : overAlternative ? 'overAlternative' : 'default'],
-                    {
-                        [styles.chipInteractiveVariants[isDarkMode ? 'dark' : 'light']]: isInteractive,
-                    }
-                )}
-                paddingLeft={paddingLeft}
-                paddingRight={12}
-                {...getPrefixedDataAttributes(dataAttributes, 'Chip')}
+            <BaseTouchable trackingEvent={props.trackingEvent} onPress={props.onPress} as="a">
+                {renderContent()}
+            </BaseTouchable>
+        );
+    }
+
+    if (props.to) {
+        return (
+            <BaseTouchable
+                trackingEvent={props.trackingEvent}
+                to={props.to}
+                fullPageOnWebView={props.fullPageOnWebView}
             >
-                {body}
-            </Box>
+                {renderContent()}
+            </BaseTouchable>
         );
     }
+
+    if (props.href) {
+        return (
+            <BaseTouchable
+                trackingEvent={props.trackingEvent}
+                href={props.href}
+                newTab={props.newTab}
+                dataAttributes={dataAttributes}
```

Comment:
why do you pass `dataAttributes` here? they are already applied in `renderContent`

Reply:
will remove